File: osm/addr-osmify-java/src/main/java/hu/vmiklos/addr_osmify/Urlopener.java
/*
 * Copyright 2019 Miklos Vajna
 *
 * SPDX-License-Identifier: MIT
 */

package hu.vmiklos.addr_osmify;

/**
 * urlopen() interface, to be implemented using HTTP or mocking.
 */
public interface Urlopener
{
    String urlopen(String urlString, String data) throws Exception;
}

/* vim:set shiftwidth=4 softtabstop=4 expandtab: */

File: osm/addr-osmify-kotlin/src/main/kotlin/Urlopener.kt
/*
 * Copyright 2020 Miklos Vajna
 *
 * SPDX-License-Identifier: MIT
 */
package hu.vmiklos.addr_osmify

/**
 * urlopen() interface, to be implemented using HTTP or mocking.
 */
interface Urlopener {
    fun urlopen(urlString: String, data: String): String
}

/* vim:set shiftwidth=4 softtabstop=4 expandtab: */
